Cron/Manager.php

```diff
@@ -56,11 +56,6 @@ protected function getReportRepo()
      */
     public function saveReports(array $reports)
     {
-        $connection = $this->manager->getConnection();
```

Comment:
@gun-romainflamant  Thanks for the proposal, this would break older implementations that need the reconnect feature, could you instead check if the method ping exists? I believe this could also solve the problem and not break older doctrine implementations.

Reply:
I understand well the problem, but leave it like that, you will loss compatibility with newer version.
Maybe we could use the method check on $connection the ping, that will check if ::ping() exist before call it, and if this method does exist the ping fonction is useless because of the autocheck of DBAL 3.0

if($connection instanceof Connection && true === method_exists($connection, 'ping') && false === $connection->ping()){